cmd/ddev/cmd/debug-refresh.go

```diff
@@ -12,12 +13,17 @@ import (
 	"github.com/spf13/cobra"
 )
 
+var (
+	buildAll bool
```

Comment:
In general, I don't like these globals for this, and prefer checking the flags as we do in most places. Of course, this is just in `ddev debug` so not important at all, so no big deal.

Reply:
The variable here is only needed because the flag has a short and long form, I just copied the logic from some other command.